apps/nl_means/process.cpp

```diff
@@ -33,13 +33,13 @@ int main(int argc, char **argv) {
             input.width(), input.height(), patch_size, search_area, sigma);
 
     // Manually-tuned version
-    double min_t_manual = benchmark(timing_iterations, 10, [&]() {
```

Comment:
Should probably just use benchmark([&]() {...}) and let it determine iterations automatically